### PR TITLE
Add publish box

### PR DESCRIPTION
**Pull request by Healthedata1**
add publlsh box include file to default layout to provide link to history file.

### CODE REVIEW

File: spec/_layouts/default.html [hunk 1]
```diff
@@ -1,13 +1,26 @@
 ---
 baseurl: "file:///Users/ehaas/Documents/FHIR/smart-app-launch-brver/output/"
```
Comment (jmandel) [inline spec/_layouts/default.html]: These paths seems to reference file locations on your own personal computer @Healthedata1  -- I wouldn't expect to see this in checked-in code.

**Comment by Healthedata1** [comment]
I will  update to what I think think the base url will be.   I can automate that before each commit.

Eric

**Comment by jmandel** [comment]
But from what you're writing, you're suggesting the file contents would need to keep changing -- that's not sustainable, since it means someone will make an error. Perhaps should be pointing to distinct files for different environments, perhaps with a command-line flag.

**Comment by Healthedata1** [comment]
I open to suggestions.

Eric

Eric M Haas, DVM, MS
Health eData Inc
1183 Mountain View Circle, Napa, 94558
707.227.2608|Skype: haas.eric1
ehaas@healthedatainc.com

**Comment by jmandel** [comment]
OK. Let's get the conflicts resolved here, and we can merge it; I'll see if I can then make some tweaks for paths.

**Comment by Healthedata1** [comment]
I removed the baseUrl in #278

**Pull request merged**